packages/common/src/ceramic-api.ts

```diff
@@ -58,24 +58,37 @@ export interface CeramicSigner extends CeramicCommon {
  * Describes Ceramic Admin API functionality
  */
 export interface AdminApi {
+  /**
+   * Generate a code needed to perform other AdminApi operations
+   */
+  generateCode(): Promise<string>
```

Comment:
This method doesn't need to be on the CeramicApi. Can be only available as an http endpoint. The ceramic-daemon can keep track of issued codes.